src/eradiate/srf_tools.py

```diff
@@ -59,32 +60,126 @@ def update_attrs(srf: xr.Dataset, filter_name: str, filter_attr: str) -> None:
     )
 
 
+def wavelength_range_width(
+    srf: t.Union[PathLike, xr.Dataset]
+) -> pint.Quantity:
+    """
+    Compute the wavelength range width of a spectral response function.
+
+    Notes
+    -----
+    The wavelength range width is defined as the difference between the upper 
+    and lower wavelength bounds.
+    """
```

Comment:
You're missing documentation for the `srf` parameter, is that intentional?

Reply:
I thought you'd tell me it's pretty obvious hence useless, so I omitted intentionally yes :smile: